### PR TITLE
cfc: auto mount share folders

### PR DESCRIPTION
**Pull request by YadongQi**
auto mount share folders for Pictures and Download

Tracked-On: OAM-97753
Signed-off-by: Yadong Qi <yadong.qi@intel.com>

### CODE REVIEW

File: groups/device-specific/caas_cfc/init.rc [hunk 1]
```diff
@@ -56,3 +56,12 @@ on post-fs
 #For OMX
 on post-fs-data
     setprop debug.stagefright.ccodec 0
+
+on boot
+    start mount_share
+
+service mount_share /vendor/bin/mount_share.sh
+    user root
```
Comment (ceiba1985) [inline groups/device-specific/caas_cfc/init.rc:64]: Please ensure cfc really don't care about the vendor binaries running as root.

File: groups/device-specific/caas_cfc/init.rc [hunk 2]
```diff
@@ -56,3 +56,18 @@ on post-fs
...
+
+service mount_share /vendor/bin/mount_share.sh
+    user system
+    group system
+    disabled
+    oneshot
+
+on property:vendor.intel.mount_share=1
+    mount 9p Download9p /data/media/0/Download mode=0777
+    mount none /data/media/0/Download /mnt/user/0/emulated/0/Download bind
```
Comment (shyjumon-n) [inline groups/device-specific/caas_cfc/init.rc:71]: we can try rbind to make reclusiveness if we expect directories created inside.

Comment (YadongQi) [reply groups/device-specific/caas_cfc/init.rc:71]: no need to rbind. rbind is to bind mountpiont recurisively. currently no sub-mountpoint under these folders.

**Review by lchen43 — APPROVED** [review]

**Review by balajim001 — APPROVED** [review]
LGTM

**Pull request closed without merging**